### PR TITLE
Feat/trigger action TmplCheckBoxComponent, TmplNumberSelectorComponent

### PR DESCRIPTION
**Pull request by egor-mir**
PR Checklist

- [x] - Latest `master` branch merged
- [x] - PR title descriptive (can be used in release notes)

## Description
Add trigger action to TmplCheckBoxComponent, TmplNumberSelectorComponent.
Refactoring styles
[NumberSelector](https://plh-teens-uat1--pr607-feat-trigger-action-0e4eb2tm.web.app/template/box_multi_1_temp)
[CheckBox](https://plh-teens-uat1--pr607-feat-trigger-action-0e4eb2tm.web.app/template/feature_checkbox)
_What this PR does_

## Git Issues

_Closes #_

## Screenshots/Videos
image
image
image

_If useful, provide screenshot or capture to highlight main changes_

### CODE REVIEW

File: src/app/shared/components/template/components/number-selector/number-selector.component.ts [hunk 1]
```diff
@@ -48,42 +50,58 @@ export class TmplNumberComponent extends TemplateBaseComponent implements ITempl
...
   increment(param: "gt" | "lt") {
-    return this.category_list ?
-      (this.checkIfContainsNextArrayItem(this.valuesFromCategoryList, param) ?
-        this.updateData(param) : null)
-      : (param === "gt" ? (this.displayValue + this.category_size <= this.max_value ?
-        (this.category_size === 1 ? this.displayValue++ : (this.displayValue += this.category_size)) : null) :
-        (this.displayValue > this.min_value - this.category_size && this.displayValue - this.category_size >= this.min_value ?
-        (this.category_size === 1 ? this.displayValue-- : this.displayValue -= this.category_size) : null));
+    this.triggerActions("changed");
+    return this.category_list
```
Comment (michaelclapham) [inline src/app/shared/components/template/components/number-selector/number-selector.component.ts:69]: I reckon if statements here would be easier to read than this many nested ternary operators

File: src/app/shared/components/template/components/number-selector/number-selector.component.ts [hunk 2]
```diff
@@ -48,42 +50,58 @@ export class TmplNumberComponent extends TemplateBaseComponent implements ITempl
...
+        ? this.category_size === 1
+          ? this.displayValue++
+          : (this.displayValue += this.category_size)
+        : null
+      : this.displayValue > this.min_value - this.category_size &&
+        this.displayValue - this.category_size >= this.min_value
+      ? this.category_size === 1
+        ? this.displayValue--
+        : (this.displayValue -= this.category_size)
+      : null;
```
Comment (michaelclapham) [inline src/app/shared/components/template/components/number-selector/number-selector.component.ts:84]: I reckon if statements here would be easier to read than this many nested ternary operators

Comment (egor-mir) [reply src/app/shared/components/template/components/number-selector/number-selector.component.ts:84]: linter made it less readable. I will refactor it on Monday in a specific branch. Issue #609

**Review by smborio — APPROVED** [review]
functional test passed

**Pull request merged**